### DISCUSSION TITLE
Adguard home and youtube

### DISCUSSION THREAD

**Discussion by Bagou450** [AdguardTeam/AdGuardHome#3210] [Q&A]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ ] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  *  Last
* **How did you install AdGuard Home:**
  *  Raspberry
* **How did you setup DNS configuration:**
  *  Pc and router
* **If it's a router or IoT, please write device model:**
  *  Freebox delta
* **CPU architecture:**
  *  AMD64
* **Operating system and version:**
  *  Windows 10

### Expected Behavior

I have add on youtube but adguard home is activate.
Same on Spotify
### Screenshots

<details><summary>Screenshot:</summary>

</details>

**Comment by Bagou450** [comment]
Oh and 

nslookup ad.doubleclick.net
Serveur :   UnKnown
Address:  192.168.0.75

Réponse ne faisant pas autorité :
Nom :    ad.doubleclick.net
Addresses:  ::
          0.0.0.0

**Comment by ameshkov** [comment]
I am not sure what the question is, but just in case, DNS is not enough to block YouTube ads.
